satpy/readers/atms_l1b_nc.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
These are not needed (we're trying to remove them from older files too)

Reply:
I have removed it.